features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,219 @@
+# Taken from the cucumber-rails project.
```

Comment:
why do you have web_steps?   you are not using cucumber - please remove any unnecessary files